src/components/chips/js/chipsDirective.js

```diff
@@ -67,11 +67,16 @@
    *    displayed when there is at least on item in the list
    * @param {boolean=} readonly Disables list manipulation (deleting or adding list items), hiding
    *    the input and delete buttons
-   * @param {expression} md-on-append An expression that when called expects you to return an object
-   *    representation of the chip input string.
+   * @param {expression} md-on-append An expression of form `myFunction($chip)` that when called
```

Comment:
Would it make more sense then to add a properly named event method for when something is added? like md-on-add? We could mark md-on-append as deprecated, giving it a better name like md-build-chip to avoid confusion.